azkaban-common/src/main/java/azkaban/utils/Emailer.java

```diff
@@ -123,6 +123,15 @@ private void sendSlaAlertEmail(final SlaOption slaOption, final String slaMessag
     }
   }
 
+  private String getJobOrFlowName(SlaOption slaOption) {
+    String jobName = (String) slaOption.getInfo().get(SlaOption.INFO_JOB_NAME);
+    if (org.apache.commons.lang.StringUtils.isNotBlank(jobName)) {
+      return jobName;
```

Comment:
maybe return flowName+"."+jobName if job name exists to be more clear?

Reply:
Will do. I had thought that it would be either-or, but apparently flow name is available in SLA info also for job level SLAs.